src/prefect/tasks/prefect/flow_run_rename.py

```diff
@@ -30,24 +31,30 @@ def __init__(
     def run(self, flow_run_id: str, flow_run_name: str) -> bool:
         """
         Args:
-            - flow_run_id (str, optional): The ID of the flow run to rename
+            - flow_run_id (str, optional): The ID of the flow run to rename. If `None`,
```

Comment:
Technically an empty string will trigger a context lookup too
```suggestion
            - flow_run_id (str, optional): The ID of the flow run to rename. If empty,
```